libs/langgraph/langgraph/prebuilt/__init__.py

```diff
@@ -1,6 +1,7 @@
 """langgraph.prebuilt exposes a higher-level API for creating and executing agents and tools."""
 
 from langgraph.prebuilt.chat_agent_executor import create_react_agent
+from langgraph.prebuilt.sequential_executor import create_sequential_executor
```

Comment:
name is weird and not super intuitive

`pipeline`? `chain`?

Reply:
yea chain might be best